spl/src/idl_build.rs

```diff
@@ -10,7 +10,7 @@ macro_rules! impl_idl_build {
         //
         // TODO: Find a better way to handle discriminators of wrapped external accounts.
         impl anchor_lang::Discriminator for $ty {
-            const DISCRIMINATOR: &[u8] = &[];
+            const DISCRIMINATOR: &'static[u8] = &[];
```

Comment:
[Format error in CI](https://github.com/coral-xyz/anchor/actions/runs/10132823075/job/28020306859?pr=3129#step:7:19)

```suggestion
            const DISCRIMINATOR: &'static [u8] = &[];
```